int32 degrees
float32 duration
bool leaveOnAtEnd
---
